src/fastoad/cmd/tests/test_api.py

```diff
@@ -101,7 +101,7 @@ def test_list_variables(cleanup):
 
 
 def test_write_n2(cleanup):
-    default_n2_file_path = pth.join(DATA_FOLDER_PATH, "n2.html")
+    default_n2_file_path = "n2.html"
```

Comment:
The test for the default location of the generated file was useful because there was some path manipulation. But now it is a mere default value, I think it is not worth testing.
I would simply remove lines 104 to 107.